lib/libutee/tee_api_objects.c

```diff
@@ -37,34 +37,52 @@ void __utee_from_attr(struct utee_attribute *ua, const TEE_Attribute *attrs,
  */
 void TEE_GetObjectInfo(TEE_ObjectHandle object, TEE_ObjectInfo *objectInfo)
 {
-	TEE_Result res;
+	struct utee_object_info info = { };
+	TEE_Result res = TEE_SUCCESS;
 
-	res = _utee_cryp_obj_get_info((unsigned long)object, objectInfo);
+	res = _utee_cryp_obj_get_info((unsigned long)object, &info);
 
 	if (res != TEE_SUCCESS)
 		TEE_Panic(res);
 
-	if (objectInfo->objectType == TEE_TYPE_CORRUPTED_OBJECT) {
+	if (info.obj_type == TEE_TYPE_CORRUPTED_OBJECT) {
 		objectInfo->keySize = 0;
 		objectInfo->maxKeySize = 0;
 		objectInfo->objectUsage = 0;
 		objectInfo->dataSize = 0;
 		objectInfo->dataPosition = 0;
 		objectInfo->handleFlags = 0;
+	} else {
+		objectInfo->objectType = info.obj_type;
+		objectInfo->keySize = info.obj_size;
+		objectInfo->maxKeySize = info.max_obj_size;
```

Comment:
How about first switching to the new names (`objectSize`/`maxObjectSize`) everywhere prior to this commit?

Reply:
I'd rather keep them as is here. These functions will need to be duplicated with a 1.1 version before we're at version 1.3.1. So it's easier to keep track of what's v1.1 if we wait with updating the new names until have copied out a version 1.1 function.

I hope it's not a big deal in the PR and it will be pretty clear what I'm meaning soon in the next PR.

Reply:
Got it, makes sense.